packages/by-name/microsoft/genpolicy/package.nix

```diff
@@ -45,6 +45,8 @@ rustPlatform.buildRustPackage rec {
       #
       # This patch is not upstreamable.
       ./0004-genpolicy-regex-check-contrast-specific-layer-src-pr.patch
+      ./0005-genpolicy-propagate-mount_options-for-empty-dirs.patch
+      ./0006-genpolicy-support-HostToContainer-mount-propagation.patch
```

Comment:
Can we add some comments here for the new patches?

Reply:
In particular what the upstream status is.